controllers/comments.controllers.js

```diff
@@ -0,0 +1,20 @@
+const { insertNewComment } = require("../models/comments.models")
+
+exports.postNewComment = (req, res, next) => {
+    const {username, body} = req.body
+    const articleId = req.params.article_id
+    if(!username || !body || !articleId){
+        return Promise.reject({status: 400, msg: "bad request"})
+        .catch((err) => {
+            next(err)
+        })
+    } else {
+    insertNewComment (username, body, articleId)
+    .then(({rows}) => {
+        res.status(201).send({comment: rows[0]})
+    })
```

Comment:
Theres a lot going on here in this controller to say a controller should just be passing data.
I'd move some of this to the model.